app/views/events/participants.html.erb

```diff
@@ -1,40 +1,66 @@
 <%- model_class = Event -%>
+<script type="text/javascript">
+  jQuery(function() {
+    $('#selectAll').click(function() {
```

Comment:
In enviroments with a lot of different scripts it is a good idea to avoid `$` as placehoder for `jQuery` or at least to define it on the functions stack. In some cases it can lead to problems.